datareduction/datareducer.cpp

```diff
@@ -35,6 +35,575 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
 {
    typedef Parameters P;
 
+   if(P::writeFullBGB) {
```

Comment:
Add a comment: "Special case, only construct all required background magnetic field datareducers, then return" or even more verbose.

Reply:
Will become obsolete with change of approach.